### ISSUE TITLE
Idea: Support for Custom moods

### ISSUE THREAD

**Issue by Paul-Lukas** [Belluxx/DaylioTools#2]
Maby all the Moods can be collected and displayed, the user would then be able to sort them to the different values manually.
What do you think about that Idea?

**Comment by Belluxx** [comment]
You mean adding a feature that allows to let the user choose a numeric value to be associated with unrecognized/custom moods?

**Comment by Paul-Lukas** [comment]
Exactly, this would allow the user to subdivide the moods finer. Let's say on a Scala of one to ten, not one to five.
Also, there would be no need to add the Languages in code, the user could do it directly in the App by them self. 

I basically know nothing about Kotlin, how would you do it?

**Comment by Belluxx** [comment]
Well it can be implemented like a custom dialog that appears when a mood is not recognized. With something like a scrollbar to choose the value.
